projects/admin-core/src/lib/application/application-form/application-form.component.ts

```diff
@@ -0,0 +1,96 @@
+import { Component, OnInit, ChangeDetectionStrategy, Input, Output, EventEmitter, OnDestroy } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { BoundsModel } from '@tailormap-viewer/api';
+import { ApplicationModel } from '@tailormap-admin/admin-api';
+import { debounceTime, filter, Subject, takeUntil } from 'rxjs';
+import { FormHelper } from '../../helpers/form.helper';
+
+const EMPTY_BOUNDS: BoundsModel = { minx: 0, miny: 0, maxx: 0, maxy: 0 };
+
+@Component({
+  selector: 'tm-admin-application-form',
+  templateUrl: './application-form.component.html',
+  styleUrls: ['./application-form.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class ApplicationFormComponent implements OnInit, OnDestroy {
+
+  private _application: ApplicationModel | null = null;
+
+  @Input()
+  public set application(application: ApplicationModel | null) {
+    this._application = application;
+    this.initForm(application);
+  }
+  public get application(): ApplicationModel | null {
+    return this._application;
+  }
+
+  @Output()
+  public updateApplication = new EventEmitter<Omit<ApplicationModel, 'id'>>();
+
+  public applicationForm = new FormGroup({
+    name: new FormControl('', { nonNullable: true, validators: [Validators.required] }),
+    title: new FormControl(''),
+    adminComments: new FormControl(''),
+    crs: new FormControl(''),
+    initialExtent: new FormControl<BoundsModel>(EMPTY_BOUNDS),
+    maxExtent: new FormControl<BoundsModel>(EMPTY_BOUNDS),
+    authenticatedRequired: new FormControl(false),
+  });
+
+  public projections = [
```

Comment:
should we use https://github.com/B3Partners/tailormap-viewer/blob/main/projects/map/src/lib/models/projection-codes.enum.ts for this?
and maybe also add the extent of the projection as a boundsmodel

Reply:
Good point, for now I'll keep it as is, because in the future we want to support more projections anyway. Currently the enum you mentioned is only used for search, because there we know in what projection the results are.